src/main/resources/db/changelog/007-add-foreignKey-to-financial_transaction.yaml

```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
```

Comment:
This changeset should by in file with creating financial transactions, no no in separate file